_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -56195,6 +56189,9 @@
 	icon_state = "1-2"
 	},
 /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,
+/obj/item/paper/safe_code{
```

Comment:
You need to remove the warden safe code from meta

Reply:
fixed